src/po/Make_mvc.mak

```
@@ -456,11 +456,14 @@ first_time: files
 	@ copy /b .\files+.\vim_to_js .\allfiles
 	set OLD_PO_FILE_INPUT=yes
 	set OLD_PO_FILE_OUTPUT=yes
-	$(XGETTEXT) --default-domain=$(LANGUAGE) --add-comments $(XGETTEXT_KEYWORDS) \
-		--files-from=.\allfiles
+	$(XGETTEXT) --default-domain=$(LANGUAGE) --add-comments \
+		$(XGETTEXT_KEYWORDS) --files-from=.\allfiles \
+		--copyright-holder="2025, The Vim Project" --package-name=Vim \
```

It's a bit strange, that `--copyright-holder` includes the year and we hard-code it. I think this would be better, what do you think:

```suggestion
		--copyright-holder="$$(date +%Y), The Vim Project" --package-name=Vim \
```

Not sure about the correct way on Windows however.

---

Sorry, this comment obviously applies to the Unix Makefile

---

This is done for the UNIX Makefile.
It's a little more complicated for MS Windows, but it's also done.

---

thank you, I'll check it out tomorrow.